contrib/colab/mnist-training.ipynb

```diff
@@ -251,7 +250,9 @@
         "\n",
         "  # Get loss function, optimizer, and model\n",
         "  device = xm.xla_device()\n",
-        "  model = MNIST().to(device)\n",
+        "  # Only instantiate model weights once in memory\n",
+        "  wrapped_model = xmp.MpModelWrapper(MNIST())\n",
```

Comment:
The model wrapper works like the serial executor. You need to create it at global scope:

https://github.com/pytorch/xla/blob/bbb4932534027b1cddc5d65db39bb82486737cda/torch_xla/distributed/xla_multiprocessing.py#L309

Reply:
Ah yes, thanks. Done.